foodsearch/src/components/pages/Recipe.js

```diff
@@ -1,14 +1,27 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { Cuisine } from "../Data/cuisine";
 export default function Resipe(){
+    // console.log(Cuisine[0]);
     return (
         <>
         <div className="display-img">
             <img src="http://res.cloudinary.com/the-infatuation/image/upload/v1671473548/Screen_Shot_2022-12-19_at_10.11.56_AM_jdwh8u.png"></img>
```

Comment:
It's not a good practice to add url directly in src, one should keep storing them in a variable.